### DISCUSSION TITLE
迅雷云盘接收不到验证码

### DISCUSSION THREAD

**Discussion by hkiceflow** [AlistGo/alist#3187] [Q&A]
卡在了验证码接收，点进去手机号不是我的，改成我的后也收不到验证码，无法获取CaptchaToken

**Comment by drw2005** [comment]
昨天按照教程我倒是挂载成功了，你手机号不是自己的很奇怪啊，再检查一下吧，一步步按教程来肯定没问题的。

不过昨晚看迅雷云盘上的视频时，感觉有些卡不知道为啥，其他比如百度、123盘都很流畅。
